controllers/nmstate_controller_test.go

```diff
@@ -305,6 +305,65 @@ var _ = Describe("NMState controller reconcile", func() {
 			Expect(anyTolerationsPresent(infraTolerations, ds.Spec.Template.Spec.Tolerations)).To(BeFalse())
 		})
 	})
+	Context("when OVS is NOT enabled", func() {
+		var (
+			request ctrl.Request
+		)
+		BeforeEach(func() {
+			s := scheme.Scheme
```

Comment:
nit: We duplicate this block of code a lot, couldn't we move the BeforeEach one level up and share this setup for both specs?
Not a must for this PR, but it wouldn't be a bad thing IMO

Reply:
It's very declarative, I prefer to do this as follow up since the issue is fixing is important for OVNKubernetes+kubevirt